ffcx/codegeneration/form.py

```diff
@@ -90,8 +90,17 @@ def generator(ir, options):
     integral_offsets = [0]
     # Note: the order of this list is defined by the enum ufcx_integral_type in ufcx.h
     for itg_type in ("cell", "exterior_facet", "interior_facet"):
-        integrals += [L.AddressOf(L.Symbol(itg)) for itg in ir.integral_names[itg_type]]
-        integral_ids += ir.subdomain_ids[itg_type]
+        unsorted_integrals = []
```

Comment:
Aren't they already sorted? Python `set()` (`representation.py` line 575)

Reply:
No, they are not. Without this, `test_jit_forms.py::test_subdomains` failed.